src/entities/trades/uniswap-v2/constants.ts

```diff
@@ -61,8 +61,8 @@ export const USDC: Record<number, Token> = {
     ChainId.ARBITRUM_ONE,
     '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8',
     6,
-    'USDC',
-    'USD//C'
+    'USDC.e',
+    'Bridged USDC'
```

Comment:
I think we should change this to Native instead of Bridged. Am I seeing something wrong here?